packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1928,6 +1933,17 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
         context: ErrorDescription('while calling onSubmitted for $action'),
       ));
     }
+
+    // Create a new TextInputConnection to replace the current one. This
+    // on iOS switches to a new input view and on Android restarts the
+    // input method. In both cases this call resets the IME.
+    //
+    // We do this for fidelity reasons, in case the developer calls
```

Comment:
avoid using `we` in comment, different people may have different interpretation of who `we` are